src/collective/behavior/banner/browser/viewlets.py

```diff
@@ -76,7 +77,8 @@ def banner(self, obj):
         if obj.banner_title:
             banner['banner_title'] = obj.banner_title
         if obj.banner_description:
-            banner['banner_description'] = obj.banner_description
+            crop = Plone(self.context, self.request).cropText
```

Comment:
The question was no about plone.api but about instanciating it directly ``Plone(self.context, self.request)`` vs.  lookup by name with (with ``getMultiAdapter`` ot ``get_name``). I guess the test-code did the first.